velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -3836,3 +3836,48 @@ TEST_F(DateTimeFunctionsTest, fromUnixtimeDouble) {
   });
   assertEqualVectors(expected, actual);
 }
+
+TEST_F(DateTimeFunctionsTest, toISO8601TestTimestamp) {
```

Comment:
Please add tests for std::nullopt input to test NULL values.

Reply:
Added.